python/tvm/relay/op/strategy/cuda.py

```diff
@@ -1210,3 +1210,16 @@ def invert_permutation_strategy_cuda(attrs, inputs, out_type, target):
         name="invert_permutation.cuda",
     )
     return strategy
+
+
+@einsum_strategy.register(["cuda", "gpu"])
+def einsum_strategy_cuda(attrs, inputs, out_type, target):
+    """einsum cuda strategy"""
+    strategy = _op.OpStrategy()
+    # TODO: Add cuda-specific op implementation for einsum
+    strategy.add_implementation(
```

Comment:
This would lead to an error if you try to run it on cuda right? It is better to remove this strategy until we have a CUDA schedule ready.

Reply:
The `test_onnx_nodes` tests run on CUDA and have passed. I will remove if you think it's necessary to remove despite that?

Reply:
Oh interesting. I didn't know that `topi.generic.schedule_extern` somehow generates a valid schedule.